suite/tests/linux/clone.c

```diff
@@ -351,10 +351,13 @@ delete_thread(pid_t pid, void *stack)
 {
```

Comment:
> iX: Test exit status of child to make sure child completed successfully #6515

style: While we put iX in the branch name, we do not put it in the issue title: if there's no issue number we just don't have a prefix.  If you look at the history of commit titles you'll see examples with no issue.  A recent one: a7e64bfd7de003d38234c0ed77a067a10ee3f65c

Reply:
Just my luck looking for examples and finding a counter-example. :-)
Text updated.
Recommend updating the docs.